apps/glusterfs/pendingop.go

```diff
@@ -28,23 +28,33 @@ import (
 type PendingOperationType int
 
 const (
+	// These values are persisted and we should never change the mapping. Even
+	// if an operation type is deprecated, use a new one and retire the number
+	// associated with it.
+
```

Comment:
Needed an empty line here because the comment was being taken as a doc for the exported const OperationUnknown.